wagtail/contrib/modeladmin/views.py

```diff
@@ -68,6 +70,7 @@ def __init__(self, model_admin):
         self.is_pagemodel = model_admin.is_pagemodel
         self.permission_helper = model_admin.permission_helper
         self.url_helper = model_admin.url_helper
+        self.locale = None
```

Comment:
not needed since you're using the `if getattr(settings, 'WAGTAIL_I18N_ENABLED', False) and issubclass(self.model, TranslatableMixin):` everywhere

Reply:
Ok!